README.md

```diff
@@ -133,6 +133,49 @@ sublime-phpcs is insanely powerful, but if you'd prefer automatic linting, [Subl
 
 Please see “[Setting up PHP CodeSniffer in Visual Studio Code](https://tommcfarlin.com/php-codesniffer-in-visual-studio-code/)”, a tutorial by Tom McFarlin.
 
+
+## Running your code through WPCS automatically using CI tools
+
+### [Travis CI](https://travis-ci.org/)
+
+To integrate PHPCS with WPCS with Travis, you'll need to install both `before_script` (or at `install`) and add the run command to the `script`.
```

Comment:
"... with Travis CI" please, also PHPCS should be added in the `before_install` section of the `.travis.yml`, this way if any errors occur the build job will report `Errored` rather than `Failed` this is important as it determines there was an issue setting up your environment rather than the code you are actually testing against with PHPCS.

So something like:  "To integrate PHPCS with WPCS with Travis CI, you'll need to install both in the `before_install` section of your `.travis.yml` configuration and add the run command to the `script`."